### DISCUSSION TITLE
does yim has protection against starting activity such as tutorial

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#2496] [General]
Someone in public session started tutorial. I saw some request control event blocked when i was in car but when i got out of it I went in tutorial.

**Comment by lonelybud** [comment]
it should have been blocked by `am_launcher::START_NEW_SCRIPT_WITH_ARGS` i guess. `RAGE_JOAAT("fm_intro")` should be the hash.

**Reply by maybegreat48** [reply]
That function *should* be blocking tutorials already

**Reply by lonelybud** [reply]
thanks. will reopen if i face the issue again.

**Comment by Rxann** [comment]
Block jobs i believe
